### PR TITLE
sknt.ru

### PR DESCRIPTION
**Pull request by galeksandrp**

### CODE REVIEW

File: src/chrome/content/rules/sknt.ru.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<!--
+sknt.ru self signed
+www.sknt.ru self signed
+feedback.sknt.ru self signed
```
Comment (J0WI) [inline src/chrome/content/rules/sknt.ru.xml]: They WFM

File: src/chrome/content/rules/sknt.ru.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+anima.sknt.ru mismatch
+ivan.sknt.ru timed out
+toxa.sknt.ru timed out
+orthodox.sknt.ru nonexist
+newfie.sknt.ru timed out
+-->
+<ruleset name="sknt.ru (partial)">
+	<target host="bill.sknt.ru" />
```
Comment (J0WI) [inline src/chrome/content/rules/sknt.ru.xml]: abonent-auth.sknt.ru

File: src/chrome/content/rules/sknt.ru.xml (hunk 1 above)
Comment (galeksandrp) [inline src/chrome/content/rules/sknt.ru.xml]: Added.

File: src/chrome/content/rules/sknt.ru.xml (hunk 2 above)
Comment (galeksandrp) [inline src/chrome/content/rules/sknt.ru.xml]: Added.

**Pull request merged**